R/bounds.r

```diff
@@ -155,56 +155,91 @@ rescale_none <- function(x, ...) {
   x
 }
 
-#' Censor any values outside of range
+#' @name oob
+#' @aliases oob_censor oob_censor_any censor oob_squish oob_squish_any squish
+#'   oob_squish_infinite squish_infinite oob_discard discard oob_keep
+#' @title Out of bounds handling
```

Comment:
Our general style is not to use `@title` — it should just be the first line of the docs